src/ApiGatewayThrottlingSettings.js

```diff
@@ -58,9 +58,6 @@ class ApiGatewayThrottlingSettings {
       }
       for (let event of functionSettings.events) {
         if (isApiGatewayEndpoint(event)) {
-          if (!hasCustomThrottlingConfig(event)) {
```

Comment:
I guess you can also remove the `hasCustomThrottlingConfig` method if it's no longer used.